app/views/case_contacts/form/details.html.erb

```diff
@@ -79,6 +79,17 @@
       </div>
     </div>
 
+    <div class="card-style-1 pl-25 mb-10">
+      <h4 class="mb-20 details__topics-label">Court report <%= "topic".pluralize(@case_contact.contact_topic_answers.count) %> <span class="content-1">(optional)</span></label></h4>
+      <div class="">
+        <%= form.fields_for(:contact_topic_answers) do |f| %>
```

Comment:
These change order when you move through and back in the form. A little confusing. Can we do alphabetical?